components/playlist/browser/resources/components/playlistItem.tsx

```diff
@@ -69,9 +69,30 @@ export default function PlaylistItem ({
   cached,
   onClick
 }: Props) {
+  const anchorElem = React.useRef<HTMLAnchorElement>(null)
```

Comment:
So, If I understand the last comment from previous PR correctly, this side effect doesn't nave to re-render React component, thus it's okay to useRef here?